src/webgpu/api/operation/rendering/indirect_draw.spec.ts

```diff
@@ -1,16 +1,10 @@
 export const description = `
 Tests for the indirect-specific aspects of drawIndirect/drawIndexedIndirect.
 
-TODO: plan and implement
-- indirect draws:
```

Comment:
invalid, destroyed, wrong usage, unaligned offset - should be validation tests, no?

Reply:
Yes, those plans cover it.